<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><title>Function reference • vegan</title><script src="../deps/jquery-3.6.0/jquery-3.6.0.min.js"></script><meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"><link href="../deps/bootstrap-5.2.2/bootstrap.min.css" rel="stylesheet"><script src="../deps/bootstrap-5.2.2/bootstrap.bundle.min.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- bootstrap-toc --><script src="https://cdn.jsdelivr.net/gh/afeld/bootstrap-toc@v1.0.1/dist/bootstrap-toc.min.js" integrity="sha256-4veVQbu7//Lk5TSmc7YV48MxtMy98e26cf5MrgZYnwo=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- search --><script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/6.4.6/fuse.js" integrity="sha512-zv6Ywkjyktsohkbp9bb45V6tEMoWhzFzXis+LrMehmJZZSys19Yxf1dopHx7WzIKxr5tK2dVcYmaCk2uqdjF4A==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/autocomplete.js/0.38.0/autocomplete.jquery.min.js" integrity="sha512-GU9ayf+66Xx2TmpxqJpliWbT5PiGYxpaG8rfnBEk1LL8l1KGkRShhngwdXK1UgqhAzWpZHSiYPc09/NwDQIGyg==" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/mark.min.js" integrity="sha512-5CYOlHXGh6QpOFA/TeTylKLWfB3ftPsde7AnmhuitiTX4K5SqCLBeKro6sPS8ilsz1Q4NRx3v8Ko2IBiszzdww==" crossorigin="anonymous"></script><!-- pkgdown --><script src="../pkgdown.js"></script><meta property="og:title" content="Function reference"><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body>
    <a href="#main" class="visually-hidden-focusable">Skip to contents</a>
    

    <nav class="navbar fixed-top navbar-dark navbar-expand-lg bg-primary"><div class="container">
    
    <a class="navbar-brand me-2" href="../index.html">vegan</a>

    <small class="nav-text text-muted me-auto" data-bs-toggle="tooltip" data-bs-placement="bottom" title="">2.6-5</small>

    
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div id="navbar" class="collapse navbar-collapse ms-3">
      <ul class="navbar-nav me-auto"><li class="active nav-item">
  <a class="nav-link" href="../reference/index.html">Reference</a>
</li>
<li class="nav-item dropdown">
  <a href="#" class="nav-link dropdown-toggle" data-bs-toggle="dropdown" role="button" aria-expanded="false" aria-haspopup="true" id="dropdown-vignettes">Vignettes</a>
  <div class="dropdown-menu" aria-labelledby="dropdown-vignettes">
    <a class="dropdown-item" href="../articles/FAQ-vegan.html">Vegan FAQ</a>
  </div>
</li>
      </ul><form class="form-inline my-2 my-lg-0" role="search">
        <input type="search" class="form-control me-sm-2" aria-label="Toggle navigation" name="search-input" data-search-index="../search.json" id="search-input" placeholder="Search for" autocomplete="off"></form>

      <ul class="navbar-nav"><li class="nav-item">
  <a class="external-link nav-link" href="https://github.com/vegandevs/vegan/" aria-label="github">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul></div>

    
  </div>
</nav><div class="container template-reference-index">
<div class="row">
  <main id="main" class="col-md-9"><div class="page-header">
      <img src="" class="logo" alt=""><h1>Function reference</h1>
    </div>

    <div class="section level2">
      <h2 id="all-functions">All functions<a class="anchor" aria-label="anchor" href="#all-functions"></a></h2>
      
      

      
    </div><div id="" class="section level2">
      
      
      

      <dl><dt>
          
          <code><a href="BCI.html">BCI</a></code> <code><a href="BCI.html">BCI.env</a></code> 
        </dt>
        <dd>Barro Colorado Island Tree Counts</dd>
      </dl><dl><dt>
          
          <code><a href="CCorA.html">CCorA()</a></code> <code><a href="CCorA.html">biplot(<i>&lt;CCorA&gt;</i>)</a></code> 
        </dt>
        <dd>Canonical Correlation Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="MDSrotate.html">MDSrotate()</a></code> 
        </dt>
        <dd>Rotate First MDS Dimension Parallel to an External Variable</dd>
      </dl><dl><dt>
          
          <code><a href="MOStest.html">MOStest()</a></code> <code><a href="MOStest.html">plot(<i>&lt;MOStest&gt;</i>)</a></code> <code><a href="MOStest.html">fieller.MOStest()</a></code> <code><a href="MOStest.html">profile(<i>&lt;MOStest&gt;</i>)</a></code> <code><a href="MOStest.html">confint(<i>&lt;MOStest&gt;</i>)</a></code> 
        </dt>
        <dd>Mitchell-Olds and Shaw Test for the Location of Quadratic Extreme</dd>
      </dl><dl><dt>
          
          <code><a href="RsquareAdj.html">RsquareAdj(<i>&lt;default&gt;</i>)</a></code> <code><a href="RsquareAdj.html">RsquareAdj(<i>&lt;rda&gt;</i>)</a></code> <code><a href="RsquareAdj.html">RsquareAdj(<i>&lt;cca&gt;</i>)</a></code> 
        </dt>
        <dd>Adjusted R-square</dd>
      </dl><dl><dt>
          
          <code><a href="SSarrhenius.html">SSarrhenius()</a></code> <code><a href="SSarrhenius.html">SSgleason()</a></code> <code><a href="SSarrhenius.html">SSgitay()</a></code> <code><a href="SSarrhenius.html">SSlomolino()</a></code> 
        </dt>
        <dd>Self-Starting nls Species-Area Models</dd>
      </dl><dl><dt>
          
          <code><a href="add1.cca.html">add1(<i>&lt;cca&gt;</i>)</a></code> <code><a href="add1.cca.html">drop1(<i>&lt;cca&gt;</i>)</a></code> 
        </dt>
        <dd>Add or Drop  Single Terms to a Constrained Ordination Model</dd>
      </dl><dl><dt>
          
          <code><a href="adipart.html">adipart()</a></code> <code><a href="adipart.html">hiersimu()</a></code> 
        </dt>
        <dd>Additive Diversity Partitioning and Hierarchical Null Model Testing</dd>
      </dl><dl><dt>
          
          <code><a href="adonis.html">adonis2()</a></code> 
        </dt>
        <dd>Permutational Multivariate Analysis of Variance Using Distance
  Matrices</dd>
      </dl><dl><dt>
          
          <code><a href="anosim.html">anosim()</a></code> 
        </dt>
        <dd>Analysis of Similarities</dd>
      </dl><dl><dt>
          
          <code><a href="anova.cca.html">anova(<i>&lt;cca&gt;</i>)</a></code> <code><a href="anova.cca.html">permutest(<i>&lt;cca&gt;</i>)</a></code> 
        </dt>
        <dd>Permutation Test for Constrained Correspondence Analysis,
  Redundancy Analysis and Constrained Analysis of Principal Coordinates</dd>
      </dl><dl><dt>
          
          <code><a href="avgdist.html">avgdist()</a></code> 
        </dt>
        <dd>Averaged Subsampled Dissimilarity Matrices</dd>
      </dl><dl><dt>
          
          <code><a href="beals.html">beals()</a></code> <code><a href="beals.html">swan()</a></code> 
        </dt>
        <dd>Beals Smoothing and Degree of Absence</dd>
      </dl><dl><dt>
          
          <code><a href="betadisper.html">betadisper()</a></code> <code><a href="betadisper.html">anova(<i>&lt;betadisper&gt;</i>)</a></code> <code><a href="betadisper.html">scores(<i>&lt;betadisper&gt;</i>)</a></code> <code><a href="betadisper.html">eigenvals(<i>&lt;betadisper&gt;</i>)</a></code> <code><a href="betadisper.html">plot(<i>&lt;betadisper&gt;</i>)</a></code> <code><a href="betadisper.html">boxplot(<i>&lt;betadisper&gt;</i>)</a></code> <code><a href="betadisper.html">TukeyHSD(<i>&lt;betadisper&gt;</i>)</a></code> <code><a href="betadisper.html">print(<i>&lt;betadisper&gt;</i>)</a></code> 
        </dt>
        <dd>Multivariate homogeneity of groups dispersions (variances)</dd>
      </dl><dl><dt>
          
          <code><a href="betadiver.html">betadiver()</a></code> <code><a href="betadiver.html">plot(<i>&lt;betadiver&gt;</i>)</a></code> <code><a href="betadiver.html">scores(<i>&lt;betadiver&gt;</i>)</a></code> 
        </dt>
        <dd>Indices of beta Diversity</dd>
      </dl><dl><dt>
          
          <code><a href="bgdispersal.html">bgdispersal()</a></code> 
        </dt>
        <dd>Coefficients of Biogeographical Dispersal Direction</dd>
      </dl><dl><dt>
          
          <code><a href="bioenv.html">bioenv(<i>&lt;default&gt;</i>)</a></code> <code><a href="bioenv.html">bioenv(<i>&lt;formula&gt;</i>)</a></code> <code><a href="bioenv.html">bioenvdist()</a></code> 
        </dt>
        <dd>Best Subset of Environmental Variables with
  Maximum (Rank) Correlation with Community Dissimilarities</dd>
      </dl><dl><dt>
          
          <code><a href="biplot.rda.html">biplot(<i>&lt;rda&gt;</i>)</a></code> 
        </dt>
        <dd>PCA biplot</dd>
      </dl><dl><dt>
          
          <code><a href="capscale.html">capscale()</a></code> <code><a href="capscale.html">dbrda()</a></code> 
        </dt>
        <dd>[Partial] Distance-based Redundancy Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="cascadeKM.html">cascadeKM()</a></code> <code><a href="cascadeKM.html">cIndexKM()</a></code> <code><a href="cascadeKM.html">plot(<i>&lt;cascadeKM&gt;</i>)</a></code> 
        </dt>
        <dd>K-means partitioning using a range of values of K</dd>
      </dl><dl><dt>
          
          <code><a href="cca.html">cca(<i>&lt;formula&gt;</i>)</a></code> <code><a href="cca.html">rda(<i>&lt;formula&gt;</i>)</a></code> <code><a href="cca.html">cca(<i>&lt;default&gt;</i>)</a></code> <code><a href="cca.html">rda(<i>&lt;default&gt;</i>)</a></code> 
        </dt>
        <dd>[Partial] [Constrained] Correspondence Analysis and Redundancy
  Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="cca.object.html">ordiYbar()</a></code> <code><a href="cca.object.html">model.frame(<i>&lt;cca&gt;</i>)</a></code> <code><a href="cca.object.html">model.matrix(<i>&lt;cca&gt;</i>)</a></code> <code><a href="cca.object.html">weights(<i>&lt;cca&gt;</i>)</a></code> 
        </dt>
        <dd>Result Object from Constrained Ordination</dd>
      </dl><dl><dt>
          
          <code><a href="clamtest.html">clamtest()</a></code> <code><a href="clamtest.html">summary(<i>&lt;clamtest&gt;</i>)</a></code> <code><a href="clamtest.html">plot(<i>&lt;clamtest&gt;</i>)</a></code> 
        </dt>
        <dd>Multinomial Species Classification Method (CLAM)</dd>
      </dl><dl><dt>
          
          <code><a href="commsim.html">commsim()</a></code> <code><a href="commsim.html">make.commsim()</a></code> <code><a href="commsim.html">print(<i>&lt;commsim&gt;</i>)</a></code> 
        </dt>
        <dd>Create an Object for Null Model Algorithms</dd>
      </dl><dl><dt>
          
          <code><a href="contribdiv.html">contribdiv()</a></code> <code><a href="contribdiv.html">plot(<i>&lt;contribdiv&gt;</i>)</a></code> 
        </dt>
        <dd>Contribution Diversity Approach</dd>
      </dl><dl><dt>
          
          <code><a href="decorana.html">decorana()</a></code> <code><a href="decorana.html">plot(<i>&lt;decorana&gt;</i>)</a></code> <code><a href="decorana.html">text(<i>&lt;decorana&gt;</i>)</a></code> <code><a href="decorana.html">points(<i>&lt;decorana&gt;</i>)</a></code> <code><a href="decorana.html">summary(<i>&lt;decorana&gt;</i>)</a></code> <code><a href="decorana.html">print(<i>&lt;summary.decorana&gt;</i>)</a></code> <code><a href="decorana.html">downweight()</a></code> <code><a href="decorana.html">scores(<i>&lt;decorana&gt;</i>)</a></code> 
        </dt>
        <dd>Detrended Correspondence Analysis and Basic Reciprocal Averaging</dd>
      </dl><dl><dt>
          
          <code><a href="decostand.html">decostand()</a></code> <code><a href="decostand.html">wisconsin()</a></code> <code><a href="decostand.html">decobackstand()</a></code> 
        </dt>
        <dd>Standardization Methods for Community Ecology</dd>
      </dl><dl><dt>
          
          <code><a href="designdist.html">designdist()</a></code> <code><a href="designdist.html">chaodist()</a></code> 
        </dt>
        <dd>Design your own Dissimilarities</dd>
      </dl><dl><dt>
          
          <code><a href="deviance.cca.html">deviance(<i>&lt;cca&gt;</i>)</a></code> <code><a href="deviance.cca.html">extractAIC(<i>&lt;cca&gt;</i>)</a></code> 
        </dt>
        <dd>Statistics Resembling Deviance and AIC for Constrained Ordination</dd>
      </dl><dl><dt>
          
          <code><a href="dispindmorisita.html">dispindmorisita()</a></code> 
        </dt>
        <dd>Morisita index of intraspecific aggregation</dd>
      </dl><dl><dt>
          
          <code><a href="dispweight.html">dispweight()</a></code> <code><a href="dispweight.html">gdispweight()</a></code> <code><a href="dispweight.html">summary(<i>&lt;dispweight&gt;</i>)</a></code> 
        </dt>
        <dd>Dispersion-based weighting of species counts</dd>
      </dl><dl><dt>
          
          <code><a href="distconnected.html">distconnected()</a></code> <code><a href="distconnected.html">no.shared()</a></code> 
        </dt>
        <dd>Connectedness of Dissimilarities</dd>
      </dl><dl><dt>
          
          <code><a href="diversity.html">diversity()</a></code> <code><a href="diversity.html">simpson.unb()</a></code> <code><a href="diversity.html">fisher.alpha()</a></code> <code><a href="diversity.html">specnumber()</a></code> 
        </dt>
        <dd>Ecological Diversity Indices</dd>
      </dl><dl><dt>
          
          <code><a href="dune.html">dune</a></code> <code><a href="dune.html">dune.env</a></code> 
        </dt>
        <dd>Vegetation and Environment in Dutch Dune Meadows.</dd>
      </dl><dl><dt>
          
          <code><a href="dune.taxon.html">dune.taxon</a></code> <code><a href="dune.taxon.html">dune.phylodis</a></code> 
        </dt>
        <dd>Taxonomic Classification and Phylogeny of Dune Meadow Species</dd>
      </dl><dl><dt>
          
          <code><a href="eigenvals.html">eigenvals()</a></code> <code><a href="eigenvals.html">summary(<i>&lt;eigenvals&gt;</i>)</a></code> 
        </dt>
        <dd>Extract Eigenvalues from an Ordination Object</dd>
      </dl><dl><dt>
          
          <code><a href="envfit.html">envfit(<i>&lt;default&gt;</i>)</a></code> <code><a href="envfit.html">envfit(<i>&lt;formula&gt;</i>)</a></code> <code><a href="envfit.html">plot(<i>&lt;envfit&gt;</i>)</a></code> <code><a href="envfit.html">scores(<i>&lt;envfit&gt;</i>)</a></code> <code><a href="envfit.html">vectorfit()</a></code> <code><a href="envfit.html">factorfit()</a></code> 
        </dt>
        <dd>Fits an Environmental Vector or Factor onto an Ordination</dd>
      </dl><dl><dt>
          
          <code><a href="eventstar.html">eventstar()</a></code> 
        </dt>
        <dd>Scale Parameter at the Minimum of the Tsallis Evenness Profile</dd>
      </dl><dl><dt>
          
          <code><a href="fisherfit.html">fisherfit()</a></code> <code><a href="fisherfit.html">prestonfit()</a></code> <code><a href="fisherfit.html">prestondistr()</a></code> <code><a href="fisherfit.html">plot(<i>&lt;prestonfit&gt;</i>)</a></code> <code><a href="fisherfit.html">lines(<i>&lt;prestonfit&gt;</i>)</a></code> <code><a href="fisherfit.html">veiledspec()</a></code> <code><a href="fisherfit.html">as.fisher()</a></code> <code><a href="fisherfit.html">plot(<i>&lt;fisher&gt;</i>)</a></code> <code><a href="fisherfit.html">as.preston()</a></code> <code><a href="fisherfit.html">plot(<i>&lt;preston&gt;</i>)</a></code> <code><a href="fisherfit.html">lines(<i>&lt;preston&gt;</i>)</a></code> 
        </dt>
        <dd>Fit Fisher's Logseries and Preston's Lognormal Model to Abundance Data</dd>
      </dl><dl><dt>
          
          <code><a href="goodness.cca.html">goodness(<i>&lt;cca&gt;</i>)</a></code> <code><a href="goodness.cca.html">inertcomp()</a></code> <code><a href="goodness.cca.html">spenvcor()</a></code> <code><a href="goodness.cca.html">intersetcor()</a></code> <code><a href="goodness.cca.html">vif.cca()</a></code> <code><a href="goodness.cca.html">alias(<i>&lt;cca&gt;</i>)</a></code> 
        </dt>
        <dd>Diagnostic Tools for [Constrained] Ordination (CCA,
  RDA, DCA, CA, PCA)</dd>
      </dl><dl><dt>
          
          <code><a href="goodness.metaMDS.html">goodness(<i>&lt;metaMDS&gt;</i>)</a></code> <code><a href="goodness.metaMDS.html">stressplot(<i>&lt;default&gt;</i>)</a></code> 
        </dt>
        <dd>Goodness of Fit and Shepard Plot for Nonmetric Multidimensional Scaling</dd>
      </dl><dl><dt>
          
          <code><a href="indpower.html">indpower()</a></code> 
        </dt>
        <dd>Indicator Power of Species</dd>
      </dl><dl><dt>
          
          <code><a href="influence.cca.html">hatvalues(<i>&lt;cca&gt;</i>)</a></code> <code><a href="influence.cca.html">rstandard(<i>&lt;cca&gt;</i>)</a></code> <code><a href="influence.cca.html">rstudent(<i>&lt;cca&gt;</i>)</a></code> <code><a href="influence.cca.html">cooks.distance(<i>&lt;cca&gt;</i>)</a></code> <code><a href="influence.cca.html">sigma(<i>&lt;cca&gt;</i>)</a></code> <code><a href="influence.cca.html">vcov(<i>&lt;cca&gt;</i>)</a></code> <code><a href="influence.cca.html">SSD(<i>&lt;cca&gt;</i>)</a></code> <code><a href="influence.cca.html">qr(<i>&lt;cca&gt;</i>)</a></code> <code><a href="influence.cca.html">df.residual(<i>&lt;cca&gt;</i>)</a></code> 
        </dt>
        <dd>Linear Model Diagnostics for Constrained Ordination</dd>
      </dl><dl><dt>
          
          <code><a href="isomap.html">isomap()</a></code> <code><a href="isomap.html">isomapdist()</a></code> <code><a href="isomap.html">summary(<i>&lt;isomap&gt;</i>)</a></code> <code><a href="isomap.html">plot(<i>&lt;isomap&gt;</i>)</a></code> 
        </dt>
        <dd>Isometric Feature Mapping Ordination</dd>
      </dl><dl><dt>
          
          <code><a href="kendall.global.html">kendall.global()</a></code> <code><a href="kendall.global.html">kendall.post()</a></code> 
        </dt>
        <dd>Kendall coefficient of concordance</dd>
      </dl><dl><dt>
          
          <code><a href="linestack.html">linestack()</a></code> 
        </dt>
        <dd>Plots One-dimensional Diagrams without Overwriting Labels</dd>
      </dl><dl><dt>
          
          <code><a href="make.cepnames.html">make.cepnames()</a></code> 
        </dt>
        <dd>Abbreviates a Botanical or Zoological Latin Name into an Eight-character Name</dd>
      </dl><dl><dt>
          
          <code><a href="mantel.html">mantel()</a></code> <code><a href="mantel.html">mantel.partial()</a></code> 
        </dt>
        <dd>Mantel and Partial Mantel Tests for Dissimilarity Matrices</dd>
      </dl><dl><dt>
          
          <code><a href="mantel.correlog.html">mantel.correlog()</a></code> <code><a href="mantel.correlog.html">plot(<i>&lt;mantel.correlog&gt;</i>)</a></code> 
        </dt>
        <dd>Mantel Correlogram</dd>
      </dl><dl><dt>
          
          <code><a href="metaMDS.html">metaMDS()</a></code> <code><a href="metaMDS.html">plot(<i>&lt;metaMDS&gt;</i>)</a></code> <code><a href="metaMDS.html">points(<i>&lt;metaMDS&gt;</i>)</a></code> <code><a href="metaMDS.html">text(<i>&lt;metaMDS&gt;</i>)</a></code> <code><a href="metaMDS.html">scores(<i>&lt;metaMDS&gt;</i>)</a></code> <code><a href="metaMDS.html">metaMDSdist()</a></code> <code><a href="metaMDS.html">metaMDSiter()</a></code> <code><a href="metaMDS.html">initMDS()</a></code> <code><a href="metaMDS.html">postMDS()</a></code> <code><a href="metaMDS.html">metaMDSredist()</a></code> 
        </dt>
        <dd>Nonmetric Multidimensional Scaling with Stable Solution from
  Random Starts, Axis Scaling and Species Scores</dd>
      </dl><dl><dt>
          
          <code><a href="mite.html">mite</a></code> <code><a href="mite.html">mite.env</a></code> <code><a href="mite.html">mite.pcnm</a></code> <code><a href="mite.html">mite.xy</a></code> 
        </dt>
        <dd>Oribatid Mite Data with Explanatory Variables</dd>
      </dl><dl><dt>
          
          <code><a href="monoMDS.html">monoMDS()</a></code> <code><a href="monoMDS.html">scores(<i>&lt;monoMDS&gt;</i>)</a></code> <code><a href="monoMDS.html">plot(<i>&lt;monoMDS&gt;</i>)</a></code> <code><a href="monoMDS.html">points(<i>&lt;monoMDS&gt;</i>)</a></code> <code><a href="monoMDS.html">text(<i>&lt;monoMDS&gt;</i>)</a></code> 
        </dt>
        <dd>Global and Local Non-metric Multidimensional Scaling and
  Linear and Hybrid Scaling</dd>
      </dl><dl><dt>
          
          <code><a href="mrpp.html">mrpp()</a></code> <code><a href="mrpp.html">meandist()</a></code> <code><a href="mrpp.html">summary(<i>&lt;meandist&gt;</i>)</a></code> <code><a href="mrpp.html">plot(<i>&lt;meandist&gt;</i>)</a></code> 
        </dt>
        <dd>Multi Response Permutation Procedure and Mean Dissimilarity Matrix</dd>
      </dl><dl><dt>
          
          <code><a href="mso.html">mso()</a></code> <code><a href="mso.html">msoplot()</a></code> 
        </dt>
        <dd>Functions for performing and displaying a spatial partitioning
of cca or rda results</dd>
      </dl><dl><dt>
          
          <code><a href="multipart.html">multipart()</a></code> 
        </dt>
        <dd>Multiplicative Diversity Partitioning</dd>
      </dl><dl><dt>
          
          <code><a href="nestedtemp.html">nestedchecker()</a></code> <code><a href="nestedtemp.html">nestedn0()</a></code> <code><a href="nestedtemp.html">nesteddisc()</a></code> <code><a href="nestedtemp.html">nestedtemp()</a></code> <code><a href="nestedtemp.html">nestednodf()</a></code> <code><a href="nestedtemp.html">nestedbetasor()</a></code> <code><a href="nestedtemp.html">nestedbetajac()</a></code> <code><a href="nestedtemp.html">plot(<i>&lt;nestedtemp&gt;</i>)</a></code> <code><a href="nestedtemp.html">plot(<i>&lt;nestednodf&gt;</i>)</a></code> 
        </dt>
        <dd>Nestedness Indices for Communities of Islands or Patches</dd>
      </dl><dl><dt>
          
          <code><a href="nobs.cca.html">nobs(<i>&lt;cca&gt;</i>)</a></code> 
        </dt>
        <dd>Extract the Number of Observations from a vegan Fit.</dd>
      </dl><dl><dt>
          
          <code><a href="nullmodel.html">nullmodel()</a></code> <code><a href="nullmodel.html">print(<i>&lt;nullmodel&gt;</i>)</a></code> <code><a href="nullmodel.html">simulate(<i>&lt;nullmodel&gt;</i>)</a></code> <code><a href="nullmodel.html">update(<i>&lt;nullmodel&gt;</i>)</a></code> <code><a href="nullmodel.html">print(<i>&lt;simmat&gt;</i>)</a></code> <code><a href="nullmodel.html">smbind()</a></code> 
        </dt>
        <dd>Null Model and Simulation</dd>
      </dl><dl><dt>
          
          <code><a href="oecosimu.html">oecosimu()</a></code> <code><a href="oecosimu.html">as.ts(<i>&lt;oecosimu&gt;</i>)</a></code> <code><a href="oecosimu.html">toCoda(<i>&lt;oecosimu&gt;</i>)</a></code> 
        </dt>
        <dd>Evaluate Statistics with Null Models of Biological Communities</dd>
      </dl><dl><dt>
          
          <code><a href="ordiArrowTextXY.html">ordiArrowTextXY()</a></code> <code><a href="ordiArrowTextXY.html">ordiArrowMul()</a></code> 
        </dt>
        <dd>Support Functions for Drawing Vectors</dd>
      </dl><dl><dt>
          
          <code><a href="ordiarrows.html">ordiarrows()</a></code> <code><a href="ordiarrows.html">ordisegments()</a></code> <code><a href="ordiarrows.html">ordigrid()</a></code> 
        </dt>
        <dd>Add Arrows and Line Segments to Ordination Diagrams</dd>
      </dl><dl><dt>
          
          <code><a href="ordihull.html">ordihull()</a></code> <code><a href="ordihull.html">ordiellipse()</a></code> <code><a href="ordihull.html">ordibar()</a></code> <code><a href="ordihull.html">ordispider()</a></code> <code><a href="ordihull.html">ordicluster()</a></code> <code><a href="ordihull.html">summary(<i>&lt;ordihull&gt;</i>)</a></code> <code><a href="ordihull.html">summary(<i>&lt;ordiellipse&gt;</i>)</a></code> <code><a href="ordihull.html">ordiareatest()</a></code> 
        </dt>
        <dd>Display Groups or Factor Levels in Ordination Diagrams</dd>
      </dl><dl><dt>
          
          <code><a href="ordilabel.html">ordilabel()</a></code> 
        </dt>
        <dd>Add Text on Non-transparent Label to an Ordination Plot.</dd>
      </dl><dl><dt>
          
          <code><a href="ordiplot.html">ordiplot()</a></code> <code><a href="ordiplot.html">identify(<i>&lt;ordiplot&gt;</i>)</a></code> <code><a href="ordiplot.html">points(<i>&lt;ordiplot&gt;</i>)</a></code> <code><a href="ordiplot.html">text(<i>&lt;ordiplot&gt;</i>)</a></code> 
        </dt>
        <dd>Alternative plot and identify Functions for Ordination</dd>
      </dl><dl><dt>
          
          <code><a href="ordipointlabel.html">ordipointlabel()</a></code> <code><a href="ordipointlabel.html">plot(<i>&lt;ordipointlabel&gt;</i>)</a></code> 
        </dt>
        <dd>Ordination Plots with Points and Optimized Locations for Text</dd>
      </dl><dl><dt>
          
          <code><a href="ordiresids.html">ordiresids()</a></code> 
        </dt>
        <dd>Plots of Residuals and Fitted Values for Constrained Ordination</dd>
      </dl><dl><dt>
          
          <code><a href="ordistep.html">ordistep()</a></code> <code><a href="ordistep.html">ordiR2step()</a></code> 
        </dt>
        <dd>Choose a Model by Permutation Tests in Constrained Ordination</dd>
      </dl><dl><dt>
          
          <code><a href="ordisurf.html">ordisurf(<i>&lt;default&gt;</i>)</a></code> <code><a href="ordisurf.html">ordisurf(<i>&lt;formula&gt;</i>)</a></code> <code><a href="ordisurf.html">calibrate(<i>&lt;ordisurf&gt;</i>)</a></code> <code><a href="ordisurf.html">plot(<i>&lt;ordisurf&gt;</i>)</a></code> 
        </dt>
        <dd>Fit and Plot Smooth Surfaces of Variables on Ordination.</dd>
      </dl><dl><dt>
          
          <code><a href="orditkplot.html">orditkplot()</a></code> <code><a href="orditkplot.html">plot(<i>&lt;orditkplot&gt;</i>)</a></code> <code><a href="orditkplot.html">points(<i>&lt;orditkplot&gt;</i>)</a></code> <code><a href="orditkplot.html">text(<i>&lt;orditkplot&gt;</i>)</a></code> <code><a href="orditkplot.html">scores(<i>&lt;orditkplot&gt;</i>)</a></code> 
        </dt>
        <dd>Ordination Plot with Movable Labels</dd>
      </dl><dl><dt>
          
          <code><a href="orditorp.html">orditorp()</a></code> 
        </dt>
        <dd>Add Text or Points to Ordination Plots</dd>
      </dl><dl><dt>
          
          <code><a href="ordixyplot.html">ordixyplot()</a></code> <code><a href="ordixyplot.html">ordisplom()</a></code> <code><a href="ordixyplot.html">ordicloud()</a></code> 
        </dt>
        <dd>Trellis (Lattice) Plots for Ordination</dd>
      </dl><dl><dt>
          
          <code><a href="pcnm.html">pcnm()</a></code> 
        </dt>
        <dd>Principal Coordinates of Neighbourhood Matrix</dd>
      </dl><dl><dt>
          
          <code><a href="permatfull.html">permatfull()</a></code> <code><a href="permatfull.html">permatswap()</a></code> <code><a href="permatfull.html">print(<i>&lt;permat&gt;</i>)</a></code> <code><a href="permatfull.html">summary(<i>&lt;permat&gt;</i>)</a></code> <code><a href="permatfull.html">print(<i>&lt;summary.permat&gt;</i>)</a></code> <code><a href="permatfull.html">plot(<i>&lt;permat&gt;</i>)</a></code> <code><a href="permatfull.html">lines(<i>&lt;permat&gt;</i>)</a></code> <code><a href="permatfull.html">as.ts(<i>&lt;permat&gt;</i>)</a></code> <code><a href="permatfull.html">toCoda(<i>&lt;permat&gt;</i>)</a></code> 
        </dt>
        <dd>Matrix Permutation Algorithms for Presence-Absence and Count Data</dd>
      </dl><dl><dt>
          
          <code><a href="permustats.html">permustats()</a></code> <code><a href="permustats.html">summary(<i>&lt;permustats&gt;</i>)</a></code> <code><a href="permustats.html">densityplot(<i>&lt;permustats&gt;</i>)</a></code> <code><a href="permustats.html">density(<i>&lt;permustats&gt;</i>)</a></code> <code><a href="permustats.html">qqnorm(<i>&lt;permustats&gt;</i>)</a></code> <code><a href="permustats.html">qqmath(<i>&lt;permustats&gt;</i>)</a></code> <code><a href="permustats.html">boxplot(<i>&lt;permustats&gt;</i>)</a></code> <code><a href="permustats.html">pairs(<i>&lt;permustats&gt;</i>)</a></code> 
        </dt>
        <dd>Extract, Analyse and Display Permutation Results</dd>
      </dl><dl><dt>
          
          <code><a href="permutations.html">permutations</a></code> 
        </dt>
        <dd>Permutation tests in Vegan</dd>
      </dl><dl><dt>
          
          <code><a href="permutest.betadisper.html">permutest(<i>&lt;betadisper&gt;</i>)</a></code> 
        </dt>
        <dd>Permutation test of multivariate homogeneity of groups dispersions
  (variances)</dd>
      </dl><dl><dt>
          
          <code><a href="plot.cca.html">plot(<i>&lt;cca&gt;</i>)</a></code> <code><a href="plot.cca.html">text(<i>&lt;cca&gt;</i>)</a></code> <code><a href="plot.cca.html">points(<i>&lt;cca&gt;</i>)</a></code> <code><a href="plot.cca.html">scores(<i>&lt;cca&gt;</i>)</a></code> <code><a href="plot.cca.html">scores(<i>&lt;rda&gt;</i>)</a></code> <code><a href="plot.cca.html">summary(<i>&lt;cca&gt;</i>)</a></code> <code><a href="plot.cca.html">print(<i>&lt;summary.cca&gt;</i>)</a></code> <code><a href="plot.cca.html">head(<i>&lt;summary.cca&gt;</i>)</a></code> <code><a href="plot.cca.html">tail(<i>&lt;summary.cca&gt;</i>)</a></code> <code><a href="plot.cca.html">labels(<i>&lt;cca&gt;</i>)</a></code> 
        </dt>
        <dd>Plot or Extract Results of Constrained Correspondence Analysis
  or Redundancy Analysis</dd>
      </dl><dl><dt>
          
          <code><a href="prc.html">prc()</a></code> <code><a href="prc.html">summary(<i>&lt;prc&gt;</i>)</a></code> <code><a href="prc.html">plot(<i>&lt;prc&gt;</i>)</a></code> 
        </dt>
        <dd>Principal Response Curves for Treatments with Repeated Observations</dd>
      </dl><dl><dt>
          
          <code><a href="predict.cca.html">fitted(<i>&lt;cca&gt;</i>)</a></code> <code><a href="predict.cca.html">fitted(<i>&lt;capscale&gt;</i>)</a></code> <code><a href="predict.cca.html">residuals(<i>&lt;cca&gt;</i>)</a></code> <code><a href="predict.cca.html">predict(<i>&lt;cca&gt;</i>)</a></code> <code><a href="predict.cca.html">predict(<i>&lt;rda&gt;</i>)</a></code> <code><a href="predict.cca.html">calibrate(<i>&lt;cca&gt;</i>)</a></code> <code><a href="predict.cca.html">coef(<i>&lt;cca&gt;</i>)</a></code> <code><a href="predict.cca.html">predict(<i>&lt;decorana&gt;</i>)</a></code> 
        </dt>
        <dd>Prediction Tools for [Constrained] Ordination (CCA,
  RDA, DCA, CA, PCA)</dd>
      </dl><dl><dt>
          
          <code><a href="procrustes.html">procrustes()</a></code> <code><a href="procrustes.html">summary(<i>&lt;procrustes&gt;</i>)</a></code> <code><a href="procrustes.html">plot(<i>&lt;procrustes&gt;</i>)</a></code> <code><a href="procrustes.html">points(<i>&lt;procrustes&gt;</i>)</a></code> <code><a href="procrustes.html">text(<i>&lt;procrustes&gt;</i>)</a></code> <code><a href="procrustes.html">lines(<i>&lt;procrustes&gt;</i>)</a></code> <code><a href="procrustes.html">residuals(<i>&lt;procrustes&gt;</i>)</a></code> <code><a href="procrustes.html">fitted(<i>&lt;procrustes&gt;</i>)</a></code> <code><a href="procrustes.html">predict(<i>&lt;procrustes&gt;</i>)</a></code> <code><a href="procrustes.html">protest()</a></code> 
        </dt>
        <dd>Procrustes Rotation of Two Configurations and PROTEST</dd>
      </dl><dl><dt>
          
          <code><a href="pyrifos.html">pyrifos</a></code> 
        </dt>
        <dd>Response of Aquatic Invertebrates to Insecticide Treatment</dd>
      </dl><dl><dt>
          
          <code><a href="radfit.html">radfit(<i>&lt;default&gt;</i>)</a></code> <code><a href="radfit.html">rad.null()</a></code> <code><a href="radfit.html">rad.preempt()</a></code> <code><a href="radfit.html">rad.lognormal()</a></code> <code><a href="radfit.html">rad.zipf()</a></code> <code><a href="radfit.html">rad.zipfbrot()</a></code> <code><a href="radfit.html">predict(<i>&lt;radline&gt;</i>)</a></code> <code><a href="radfit.html">plot(<i>&lt;radfit&gt;</i>)</a></code> <code><a href="radfit.html">plot(<i>&lt;radfit.frame&gt;</i>)</a></code> <code><a href="radfit.html">plot(<i>&lt;radline&gt;</i>)</a></code> <code><a href="radfit.html">radlattice()</a></code> <code><a href="radfit.html">lines(<i>&lt;radfit&gt;</i>)</a></code> <code><a href="radfit.html">points(<i>&lt;radfit&gt;</i>)</a></code> <code><a href="radfit.html">as.rad()</a></code> <code><a href="radfit.html">plot(<i>&lt;rad&gt;</i>)</a></code> 
        </dt>
        <dd>Rank -- Abundance or Dominance / Diversity Models</dd>
      </dl><dl><dt>
          
          <code><a href="rankindex.html">rankindex()</a></code> 
        </dt>
        <dd>Compares Dissimilarity Indices for Gradient Detection</dd>
      </dl><dl><dt>
          
          <code><a href="rarefy.html">rarefy()</a></code> <code><a href="rarefy.html">rrarefy()</a></code> <code><a href="rarefy.html">drarefy()</a></code> <code><a href="rarefy.html">rarecurve()</a></code> <code><a href="rarefy.html">rareslope()</a></code> 
        </dt>
        <dd>Rarefaction Species Richness</dd>
      </dl><dl><dt>
          
          <code><a href="raupcrick.html">raupcrick()</a></code> 
        </dt>
        <dd>Raup-Crick Dissimilarity with Unequal Sampling Densities of Species</dd>
      </dl><dl><dt>
          
          <code><a href="read.cep.html">read.cep()</a></code> 
        </dt>
        <dd>Reads a CEP (Canoco) data file</dd>
      </dl><dl><dt>
          
          <code><a href="renyi.html">renyi()</a></code> <code><a href="renyi.html">plot(<i>&lt;renyi&gt;</i>)</a></code> <code><a href="renyi.html">renyiaccum()</a></code> <code><a href="renyi.html">plot(<i>&lt;renyiaccum&gt;</i>)</a></code> <code><a href="renyi.html">persp(<i>&lt;renyiaccum&gt;</i>)</a></code> 
        </dt>
        <dd>Renyi and Hill Diversities and Corresponding Accumulation Curves</dd>
      </dl><dl><dt>
          
          <code><a href="reorder.hclust.html">reorder(<i>&lt;hclust&gt;</i>)</a></code> <code><a href="reorder.hclust.html">rev(<i>&lt;hclust&gt;</i>)</a></code> <code><a href="reorder.hclust.html">scores(<i>&lt;hclust&gt;</i>)</a></code> <code><a href="reorder.hclust.html">cutreeord()</a></code> 
        </dt>
        <dd>Reorder a Hierarchical Clustering Tree</dd>
      </dl><dl><dt>
          
          <code><a href="scores.html">scores(<i>&lt;default&gt;</i>)</a></code> 
        </dt>
        <dd>Get Species or Site Scores from an Ordination</dd>
      </dl><dl><dt>
          
          <code><a href="screeplot.cca.html">screeplot(<i>&lt;cca&gt;</i>)</a></code> <code><a href="screeplot.cca.html">screeplot(<i>&lt;decorana&gt;</i>)</a></code> <code><a href="screeplot.cca.html">screeplot(<i>&lt;prcomp&gt;</i>)</a></code> <code><a href="screeplot.cca.html">screeplot(<i>&lt;princomp&gt;</i>)</a></code> <code><a href="screeplot.cca.html">bstick()</a></code> 
        </dt>
        <dd>Screeplots for Ordination Results and Broken Stick Distributions</dd>
      </dl><dl><dt>
          
          <code><a href="simper.html">simper()</a></code> <code><a href="simper.html">summary(<i>&lt;simper&gt;</i>)</a></code> 
        </dt>
        <dd>Similarity Percentages</dd>
      </dl><dl><dt>
          
          <code><a href="simulate.rda.html">simulate(<i>&lt;rda&gt;</i>)</a></code> 
        </dt>
        <dd>Simulate Responses with Gaussian Error or Permuted Residuals for Constrained Ordination</dd>
      </dl><dl><dt>
          
          <code><a href="sipoo.html">sipoo</a></code> <code><a href="sipoo.html">sipoo.map</a></code> 
        </dt>
        <dd>Birds in the Archipelago of Sipoo (Sibbo and Borgå)</dd>
      </dl><dl><dt>
          
          <code><a href="spantree.html">spantree()</a></code> <code><a href="spantree.html">as.hclust(<i>&lt;spantree&gt;</i>)</a></code> <code><a href="spantree.html">cophenetic(<i>&lt;spantree&gt;</i>)</a></code> <code><a href="spantree.html">spandepth()</a></code> <code><a href="spantree.html">plot(<i>&lt;spantree&gt;</i>)</a></code> <code><a href="spantree.html">lines(<i>&lt;spantree&gt;</i>)</a></code> 
        </dt>
        <dd>Minimum Spanning Tree</dd>
      </dl><dl><dt>
          
          <code><a href="specaccum.html">specaccum()</a></code> <code><a href="specaccum.html">plot(<i>&lt;specaccum&gt;</i>)</a></code> <code><a href="specaccum.html">boxplot(<i>&lt;specaccum&gt;</i>)</a></code> <code><a href="specaccum.html">fitspecaccum()</a></code> <code><a href="specaccum.html">plot(<i>&lt;fitspecaccum&gt;</i>)</a></code> <code><a href="specaccum.html">predict(<i>&lt;specaccum&gt;</i>)</a></code> <code><a href="specaccum.html">predict(<i>&lt;fitspecaccum&gt;</i>)</a></code> <code><a href="specaccum.html">specslope()</a></code> 
        </dt>
        <dd>Species Accumulation Curves</dd>
      </dl><dl><dt>
          
          <code><a href="specpool.html">specpool()</a></code> <code><a href="specpool.html">estimateR()</a></code> <code><a href="specpool.html">specpool2vect()</a></code> <code><a href="specpool.html">poolaccum()</a></code> <code><a href="specpool.html">estaccumR()</a></code> <code><a href="specpool.html">summary(<i>&lt;poolaccum&gt;</i>)</a></code> <code><a href="specpool.html">plot(<i>&lt;poolaccum&gt;</i>)</a></code> 
        </dt>
        <dd>Extrapolated Species Richness in a Species Pool</dd>
      </dl><dl><dt>
          
          <code><a href="sppscores.html">`sppscores&lt;-`()</a></code> 
        </dt>
        <dd>Add or Replace Species Scores in Distance-Based Ordination</dd>
      </dl><dl><dt>
          
          <code><a href="stepacross.html">stepacross()</a></code> 
        </dt>
        <dd>Stepacross as Flexible Shortest Paths or Extended Dissimilarities</dd>
      </dl><dl><dt>
          
          <code><a href="stressplot.wcmdscale.html">stressplot(<i>&lt;wcmdscale&gt;</i>)</a></code> 
        </dt>
        <dd>Display Ordination Distances Against Observed Distances in Eigenvector Ordinations</dd>
      </dl><dl><dt>
          
          <code><a href="taxondive.html">taxondive()</a></code> <code><a href="taxondive.html">taxa2dist()</a></code> 
        </dt>
        <dd>Indices of Taxonomic Diversity and Distinctness</dd>
      </dl><dl><dt>
          
          <code><a href="tolerance.html">tolerance()</a></code> 
        </dt>
        <dd>Species tolerances and sample heterogeneities</dd>
      </dl><dl><dt>
          
          <code><a href="treedive.html">treedive()</a></code> <code><a href="treedive.html">treeheight()</a></code> <code><a href="treedive.html">treedist()</a></code> 
        </dt>
        <dd>Functional Diversity and Community Distances from Species Trees</dd>
      </dl><dl><dt>
          
          <code><a href="tsallis.html">tsallis()</a></code> <code><a href="tsallis.html">tsallisaccum()</a></code> <code><a href="tsallis.html">persp(<i>&lt;tsallisaccum&gt;</i>)</a></code> 
        </dt>
        <dd>Tsallis Diversity and Corresponding Accumulation Curves</dd>
      </dl><dl><dt>
          
          <code><a href="varechem.html">varechem</a></code> <code><a href="varechem.html">varespec</a></code> 
        </dt>
        <dd>Vegetation and environment in lichen pastures</dd>
      </dl><dl><dt>
          
          <code><a href="varpart.html">varpart()</a></code> <code><a href="varpart.html">summary(<i>&lt;varpart&gt;</i>)</a></code> <code><a href="varpart.html">showvarparts()</a></code> <code><a href="varpart.html">plot(<i>&lt;varpart234&gt;</i>)</a></code> 
        </dt>
        <dd>Partition the Variation of Community Matrix by 2, 3, or 4 Explanatory Matrices</dd>
      </dl><dl><dt>
          
          <code><a href="vegan-deprecated.html">adonis()</a></code> <code><a href="vegan-deprecated.html">as.mcmc.oecosimu()</a></code> <code><a href="vegan-deprecated.html">as.mcmc.permat()</a></code> 
        </dt>
        <dd>Deprecated Functions in vegan package</dd>
      </dl><dl><dt>
          
          <code><a href="vegan-internal.html">ordiParseFormula()</a></code> <code><a href="vegan-internal.html">ordiTerminfo()</a></code> <code><a href="vegan-internal.html">ordiNAexclude()</a></code> <code><a href="vegan-internal.html">ordiNApredict()</a></code> <code><a href="vegan-internal.html">ordiArgAbsorber()</a></code> <code><a href="vegan-internal.html">centroids.cca()</a></code> <code><a href="vegan-internal.html">getPermuteMatrix()</a></code> <code><a href="vegan-internal.html">howHead()</a></code> <code><a href="vegan-internal.html">pasteCall()</a></code> <code><a href="vegan-internal.html">veganCovEllipse()</a></code> <code><a href="vegan-internal.html">veganMahatrans()</a></code> <code><a href="vegan-internal.html">hierParseFormula()</a></code> <code><a href="vegan-internal.html">GowerDblcen()</a></code> <code><a href="vegan-internal.html">addLingoes()</a></code> <code><a href="vegan-internal.html">addCailliez()</a></code> 
        </dt>
        <dd>Internal vegan functions</dd>
      </dl><dl><dt>
          
          <code><a href="vegan-package.html">vegan-package</a></code> <code><a href="vegan-package.html">vegan</a></code> 
        </dt>
        <dd>Community Ecology Package: Ordination, Diversity and Dissimilarities</dd>
      </dl><dl><dt>
          
          <code><a href="vegdist.html">vegdist()</a></code> 
        </dt>
        <dd>Dissimilarity Indices for Community Ecologists</dd>
      </dl><dl><dt>
          
          <code><a href="vegemite.html">vegemite()</a></code> <code><a href="vegemite.html">tabasco()</a></code> <code><a href="vegemite.html">coverscale()</a></code> 
        </dt>
        <dd>Display Compact Ordered Community Tables</dd>
      </dl><dl><dt>
          
          <code><a href="wascores.html">wascores()</a></code> <code><a href="wascores.html">eigengrad()</a></code> 
        </dt>
        <dd>Weighted Averages Scores for Species</dd>
      </dl><dl><dt>
          
          <code><a href="wcmdscale.html">wcmdscale()</a></code> <code><a href="wcmdscale.html">plot(<i>&lt;wcmdscale&gt;</i>)</a></code> <code><a href="wcmdscale.html">scores(<i>&lt;wcmdscale&gt;</i>)</a></code> 
        </dt>
        <dd>Weighted Classical (Metric) Multidimensional Scaling</dd>
      </dl></div>
  </main></div>


    <footer><div class="pkgdown-footer-left">
  <p></p><p>Developed by Jari Oksanen, Gavin L. Simpson, F. Guillaume Blanchet, Roeland Kindt, Pierre Legendre, Peter R. Minchin, R.B. O’Hara, Peter Solymos, M. Henry H. Stevens, Eduard Szoecs, Helene Wagner, Matt Barbour, Michael Bedward, Ben Bolker, Daniel Borcard, Gustavo Carvalho, Michael Chirico, Miquel De Caceres, Sebastien Durand, Heloisa Beatriz Antoniazi Evangelista, Rich FitzJohn, Michael Friendly, Brendan Furneaux, Geoffrey Hannigan, Mark O. Hill, Leo Lahti, Dan McGlinn, Marie-Helene Ouellette, Eduardo Ribeiro Cunha, Tyler Smith, Adrian Stier, Cajo J.F. Ter Braak, James Weedon.</p>
</div>

<div class="pkgdown-footer-right">
  <p></p><p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.7.</p>
</div>

    </footer></div>

  

  

  </body></html>

