<!-- Generated by pkgdown: do not edit by hand -->
<!DOCTYPE html>
<html lang="en">
  <head>
  <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">

<title>Function reference • vegan</title>


<!-- jquery -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
<!-- Bootstrap -->

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha256-916EbMg70RQy9LHiGkXzG8hSg9EdNy97GazNG/aiY1w=" crossorigin="anonymous" />

<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script>

<!-- Font Awesome icons -->
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.7.1/css/all.min.css" integrity="sha256-nAmazAk6vS34Xqo0BSrTb+abbtFlgsFK7NKSi6o7Y78=" crossorigin="anonymous" />
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.7.1/css/v4-shims.min.css" integrity="sha256-6qHlizsOWFskGlwVOKuns+D1nB6ssZrHQrNj1wGplHc=" crossorigin="anonymous" />

<!-- clipboard.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.4/clipboard.min.js" integrity="sha256-FiZwavyI2V6+EXO1U+xzLG3IKldpiTFf3153ea9zikQ=" crossorigin="anonymous"></script>

<!-- headroom.js -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.9.4/headroom.min.js" integrity="sha256-DJFC1kqIhelURkuza0AvYal5RxMtpzLjFhsnVIeuk+U=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.9.4/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script>

<!-- pkgdown -->
<link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script>




<meta property="og:title" content="Function reference" />




<!-- mathjax -->
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script>

<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->



  </head>

  <body>
    <div class="container template-reference-index">
      <header>
      <div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">vegan</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">2.6.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="../index.html">
    <span class="fas fa fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="../articles/FAQ-vegan.html">UNKNOWN TITLE</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/vegandevs/vegan">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
      
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header>

<div class="row">
  <div class="contents col-md-9">
    <div class="page-header">
      <h1>Reference</h1>
    </div>

    <table class="ref-index">

    <colgroup>
      
      <col class="alias" />
      <col class="title" />
    </colgroup>

    <tbody>
      <tr>
        <th colspan="2">
          <h2 id="section-all-functions" class="hasAnchor"><a href="#section-all-functions" class="anchor"></a>All functions</h2>
          <p class="section-desc"></p>
        </th>
      </tr>
      <tr>
        
        <td>
          <p><code><a href="BCI.html">BCI</a></code> <code><a href="BCI.html">BCI.env</a></code> </p>
        </td>
        <td><p>Barro Colorado Island Tree Counts</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="CCorA.html">CCorA()</a></code> <code><a href="CCorA.html">biplot(<i>&lt;CCorA&gt;</i>)</a></code> </p>
        </td>
        <td><p>Canonical Correlation Analysis</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="MDSrotate.html">MDSrotate()</a></code> </p>
        </td>
        <td><p>Rotate First MDS Dimension Parallel to an External Variable</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="MOStest.html">MOStest()</a></code> <code><a href="MOStest.html">plot(<i>&lt;MOStest&gt;</i>)</a></code> <code><a href="MOStest.html">fieller.MOStest()</a></code> <code><a href="MOStest.html">profile(<i>&lt;MOStest&gt;</i>)</a></code> <code><a href="MOStest.html">confint(<i>&lt;MOStest&gt;</i>)</a></code> </p>
        </td>
        <td><p>Mitchell-Olds \&amp; Shaw Test for the Location of Quadratic Extreme</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="RsquareAdj.html">RsquareAdj(<i>&lt;default&gt;</i>)</a></code> <code><a href="RsquareAdj.html">RsquareAdj(<i>&lt;rda&gt;</i>)</a></code> <code><a href="RsquareAdj.html">RsquareAdj(<i>&lt;cca&gt;</i>)</a></code> </p>
        </td>
        <td><p>Adjusted R-square</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="SSarrhenius.html">SSarrhenius()</a></code> <code><a href="SSarrhenius.html">SSgleason()</a></code> <code><a href="SSarrhenius.html">SSgitay()</a></code> <code><a href="SSarrhenius.html">SSlomolino()</a></code> </p>
        </td>
        <td><p>Self-Starting nls Species-Area Models</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="add1.cca.html">add1(<i>&lt;cca&gt;</i>)</a></code> <code><a href="add1.cca.html">drop1(<i>&lt;cca&gt;</i>)</a></code> </p>
        </td>
        <td><p>Add or Drop  Single Terms to a Constrained Ordination Model</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="adipart.html">adipart()</a></code> <code><a href="adipart.html">hiersimu()</a></code> </p>
        </td>
        <td><p>Additive Diversity Partitioning and Hierarchical Null Model Testing</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="adonis.html">adonis()</a></code> </p>
        </td>
        <td><p>Permutational Multivariate Analysis of Variance Using Distance Matrices</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="anosim.html">anosim()</a></code> </p>
        </td>
        <td><p>Analysis of Similarities</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="anova.cca.html">anova(<i>&lt;cca&gt;</i>)</a></code> <code><a href="anova.cca.html">permutest(<i>&lt;cca&gt;</i>)</a></code> </p>
        </td>
        <td><p>Permutation Test for Constrained Correspondence Analysis,
  Redundancy Analysis and Constrained Analysis of Principal Coordinates</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="avgdist.html">avgdist()</a></code> </p>
        </td>
        <td><p>Averaged Subsampled Dissimilarity Matrices</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="beals.html">beals()</a></code> <code><a href="beals.html">swan()</a></code> </p>
        </td>
        <td><p>Beals Smoothing and Degree of Absence</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="betadisper.html">betadisper()</a></code> <code><a href="betadisper.html">anova(<i>&lt;betadisper&gt;</i>)</a></code> <code><a href="betadisper.html">scores(<i>&lt;betadisper&gt;</i>)</a></code> <code><a href="betadisper.html">eigenvals(<i>&lt;betadisper&gt;</i>)</a></code> <code><a href="betadisper.html">plot(<i>&lt;betadisper&gt;</i>)</a></code> <code><a href="betadisper.html">boxplot(<i>&lt;betadisper&gt;</i>)</a></code> <code><a href="betadisper.html">TukeyHSD(<i>&lt;betadisper&gt;</i>)</a></code> <code><a href="betadisper.html">print(<i>&lt;betadisper&gt;</i>)</a></code> </p>
        </td>
        <td><p>Multivariate homogeneity of groups dispersions (variances)</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="betadiver.html">betadiver()</a></code> <code><a href="betadiver.html">plot(<i>&lt;betadiver&gt;</i>)</a></code> <code><a href="betadiver.html">scores(<i>&lt;betadiver&gt;</i>)</a></code> </p>
        </td>
        <td><p>Indices of beta Diversity</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="bgdispersal.html">bgdispersal()</a></code> </p>
        </td>
        <td><p>Coefficients of Biogeographical Dispersal Direction</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="bioenv.html">bioenv(<i>&lt;default&gt;</i>)</a></code> <code><a href="bioenv.html">bioenv(<i>&lt;formula&gt;</i>)</a></code> <code><a href="bioenv.html">bioenvdist()</a></code> </p>
        </td>
        <td><p>Best Subset of Environmental Variables with
  Maximum (Rank) Correlation with Community Dissimilarities</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="biplot.rda.html">biplot(<i>&lt;rda&gt;</i>)</a></code> </p>
        </td>
        <td><p>PCA biplot</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="capscale.html">capscale()</a></code> <code><a href="capscale.html">dbrda()</a></code> </p>
        </td>
        <td><p>[Partial] Distance-based Redundancy Analysis</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="cascadeKM.html">cascadeKM()</a></code> <code><a href="cascadeKM.html">cIndexKM()</a></code> <code><a href="cascadeKM.html">plot(<i>&lt;cascadeKM&gt;</i>)</a></code> </p>
        </td>
        <td><p>K-means partitioning using a range of values of K</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="cca.html">cca(<i>&lt;formula&gt;</i>)</a></code> <code><a href="cca.html">rda(<i>&lt;formula&gt;</i>)</a></code> <code><a href="cca.html">cca(<i>&lt;default&gt;</i>)</a></code> <code><a href="cca.html">rda(<i>&lt;default&gt;</i>)</a></code> </p>
        </td>
        <td><p>[Partial] [Constrained] Correspondence Analysis and Redundancy
  Analysis</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="cca.object.html">ordiYbar()</a></code> <code><a href="cca.object.html">model.frame(<i>&lt;cca&gt;</i>)</a></code> <code><a href="cca.object.html">model.matrix(<i>&lt;cca&gt;</i>)</a></code> <code><a href="cca.object.html">weights(<i>&lt;cca&gt;</i>)</a></code> </p>
        </td>
        <td><p>Result Object from Constrained Ordination</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="clamtest.html">clamtest()</a></code> <code><a href="clamtest.html">summary(<i>&lt;clamtest&gt;</i>)</a></code> <code><a href="clamtest.html">plot(<i>&lt;clamtest&gt;</i>)</a></code> </p>
        </td>
        <td><p>Multinomial Species Classification Method (CLAM)</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="commsim.html">commsim()</a></code> <code><a href="commsim.html">make.commsim()</a></code> <code><a href="commsim.html">print(<i>&lt;commsim&gt;</i>)</a></code> </p>
        </td>
        <td><p>Create an Object for Null Model Algorithms</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="contribdiv.html">contribdiv()</a></code> <code><a href="contribdiv.html">plot(<i>&lt;contribdiv&gt;</i>)</a></code> </p>
        </td>
        <td><p>Contribution Diversity Approach</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="decorana.html">decorana()</a></code> <code><a href="decorana.html">plot(<i>&lt;decorana&gt;</i>)</a></code> <code><a href="decorana.html">text(<i>&lt;decorana&gt;</i>)</a></code> <code><a href="decorana.html">points(<i>&lt;decorana&gt;</i>)</a></code> <code><a href="decorana.html">summary(<i>&lt;decorana&gt;</i>)</a></code> <code><a href="decorana.html">print(<i>&lt;summary.decorana&gt;</i>)</a></code> <code><a href="decorana.html">downweight()</a></code> <code><a href="decorana.html">scores(<i>&lt;decorana&gt;</i>)</a></code> </p>
        </td>
        <td><p>Detrended Correspondence Analysis and Basic Reciprocal Averaging</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="decostand.html">decostand()</a></code> <code><a href="decostand.html">wisconsin()</a></code> </p>
        </td>
        <td><p>Standardization Methods for Community Ecology</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="designdist.html">designdist()</a></code> <code><a href="designdist.html">chaodist()</a></code> </p>
        </td>
        <td><p>Design your own Dissimilarities</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="deviance.cca.html">deviance(<i>&lt;cca&gt;</i>)</a></code> <code><a href="deviance.cca.html">extractAIC(<i>&lt;cca&gt;</i>)</a></code> </p>
        </td>
        <td><p>Statistics Resembling Deviance and AIC for Constrained Ordination</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="dispindmorisita.html">dispindmorisita()</a></code> </p>
        </td>
        <td><p>Morisita index of intraspecific aggregation</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="dispweight.html">dispweight()</a></code> <code><a href="dispweight.html">gdispweight()</a></code> <code><a href="dispweight.html">summary(<i>&lt;dispweight&gt;</i>)</a></code> </p>
        </td>
        <td><p>Dispersion-based weighting of species counts</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="distconnected.html">distconnected()</a></code> <code><a href="distconnected.html">no.shared()</a></code> </p>
        </td>
        <td><p>Connectedness of Dissimilarities</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="diversity.html">diversity()</a></code> <code><a href="diversity.html">fisher.alpha()</a></code> <code><a href="diversity.html">specnumber()</a></code> </p>
        </td>
        <td><p>Ecological Diversity Indices</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="dune.html">dune</a></code> <code><a href="dune.html">dune.env</a></code> </p>
        </td>
        <td><p>Vegetation and Environment in Dutch Dune Meadows.</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="dune.taxon.html">dune.taxon</a></code> <code><a href="dune.taxon.html">dune.phylodis</a></code> </p>
        </td>
        <td><p>Taxonomic Classification and Phylogeny of Dune Meadow Species</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="eigenvals.html">eigenvals()</a></code> <code><a href="eigenvals.html">summary(<i>&lt;eigenvals&gt;</i>)</a></code> </p>
        </td>
        <td><p>Extract Eigenvalues from an Ordination Object</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="envfit.html">envfit(<i>&lt;default&gt;</i>)</a></code> <code><a href="envfit.html">envfit(<i>&lt;formula&gt;</i>)</a></code> <code><a href="envfit.html">plot(<i>&lt;envfit&gt;</i>)</a></code> <code><a href="envfit.html">scores(<i>&lt;envfit&gt;</i>)</a></code> <code><a href="envfit.html">vectorfit()</a></code> <code><a href="envfit.html">factorfit()</a></code> </p>
        </td>
        <td><p>Fits an Environmental Vector or Factor onto an Ordination</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="eventstar.html">eventstar()</a></code> </p>
        </td>
        <td><p>Scale Parameter at the Minimum of the Tsallis Evenness Profile</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="fisherfit.html">fisherfit()</a></code> <code><a href="fisherfit.html">prestonfit()</a></code> <code><a href="fisherfit.html">prestondistr()</a></code> <code><a href="fisherfit.html">plot(<i>&lt;prestonfit&gt;</i>)</a></code> <code><a href="fisherfit.html">lines(<i>&lt;prestonfit&gt;</i>)</a></code> <code><a href="fisherfit.html">veiledspec()</a></code> <code><a href="fisherfit.html">as.fisher()</a></code> <code><a href="fisherfit.html">plot(<i>&lt;fisher&gt;</i>)</a></code> <code><a href="fisherfit.html">as.preston()</a></code> <code><a href="fisherfit.html">plot(<i>&lt;preston&gt;</i>)</a></code> <code><a href="fisherfit.html">lines(<i>&lt;preston&gt;</i>)</a></code> </p>
        </td>
        <td><p>Fit Fisher's Logseries and Preston's Lognormal Model to Abundance Data</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="goodness.cca.html">goodness(<i>&lt;cca&gt;</i>)</a></code> <code><a href="goodness.cca.html">inertcomp()</a></code> <code><a href="goodness.cca.html">spenvcor()</a></code> <code><a href="goodness.cca.html">intersetcor()</a></code> <code><a href="goodness.cca.html">vif.cca()</a></code> <code><a href="goodness.cca.html">alias(<i>&lt;cca&gt;</i>)</a></code> </p>
        </td>
        <td><p>Diagnostic Tools for [Constrained] Ordination (CCA,
  RDA, DCA, CA, PCA)</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="goodness.metaMDS.html">goodness(<i>&lt;metaMDS&gt;</i>)</a></code> <code><a href="goodness.metaMDS.html">stressplot(<i>&lt;default&gt;</i>)</a></code> </p>
        </td>
        <td><p>Goodness of Fit and Shepard Plot for Nonmetric Multidimensional Scaling</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="indpower.html">indpower()</a></code> </p>
        </td>
        <td><p>Indicator Power of Species</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="influence.cca.html">hatvalues(<i>&lt;cca&gt;</i>)</a></code> <code><a href="influence.cca.html">rstandard(<i>&lt;cca&gt;</i>)</a></code> <code><a href="influence.cca.html">rstudent(<i>&lt;cca&gt;</i>)</a></code> <code><a href="influence.cca.html">cooks.distance(<i>&lt;cca&gt;</i>)</a></code> <code><a href="influence.cca.html">sigma(<i>&lt;cca&gt;</i>)</a></code> <code><a href="influence.cca.html">vcov(<i>&lt;cca&gt;</i>)</a></code> <code><a href="influence.cca.html">SSD(<i>&lt;cca&gt;</i>)</a></code> <code><a href="influence.cca.html">qr(<i>&lt;cca&gt;</i>)</a></code> <code><a href="influence.cca.html">df.residual(<i>&lt;cca&gt;</i>)</a></code> </p>
        </td>
        <td><p>Linear Model Diagnostics for Constrained Ordination</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="isomap.html">isomap()</a></code> <code><a href="isomap.html">isomapdist()</a></code> <code><a href="isomap.html">summary(<i>&lt;isomap&gt;</i>)</a></code> <code><a href="isomap.html">plot(<i>&lt;isomap&gt;</i>)</a></code> </p>
        </td>
        <td><p>Isometric Feature Mapping Ordination</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="kendall.global.html">kendall.global()</a></code> <code><a href="kendall.global.html">kendall.post()</a></code> </p>
        </td>
        <td><p>Kendall coefficient of concordance</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="linestack.html">linestack()</a></code> </p>
        </td>
        <td><p>Plots One-dimensional Diagrams without Overwriting Labels</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="make.cepnames.html">make.cepnames()</a></code> </p>
        </td>
        <td><p>Abbreviates a Botanical or Zoological Latin Name into an Eight-character Name</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="mantel.html">mantel()</a></code> <code><a href="mantel.html">mantel.partial()</a></code> </p>
        </td>
        <td><p>Mantel and Partial Mantel Tests for Dissimilarity Matrices</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="mantel.correlog.html">mantel.correlog()</a></code> <code><a href="mantel.correlog.html">plot(<i>&lt;mantel.correlog&gt;</i>)</a></code> </p>
        </td>
        <td><p>Mantel Correlogram</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="metaMDS.html">metaMDS()</a></code> <code><a href="metaMDS.html">plot(<i>&lt;metaMDS&gt;</i>)</a></code> <code><a href="metaMDS.html">points(<i>&lt;metaMDS&gt;</i>)</a></code> <code><a href="metaMDS.html">text(<i>&lt;metaMDS&gt;</i>)</a></code> <code><a href="metaMDS.html">scores(<i>&lt;metaMDS&gt;</i>)</a></code> <code><a href="metaMDS.html">metaMDSdist()</a></code> <code><a href="metaMDS.html">metaMDSiter()</a></code> <code><a href="metaMDS.html">initMDS()</a></code> <code><a href="metaMDS.html">postMDS()</a></code> <code><a href="metaMDS.html">metaMDSredist()</a></code> </p>
        </td>
        <td><p>Nonmetric Multidimensional Scaling with Stable Solution from
  Random Starts, Axis Scaling and Species Scores</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="mite.html">mite</a></code> <code><a href="mite.html">mite.env</a></code> <code><a href="mite.html">mite.pcnm</a></code> <code><a href="mite.html">mite.xy</a></code> </p>
        </td>
        <td><p>Oribatid Mite Data with Explanatory Variables</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="monoMDS.html">monoMDS()</a></code> <code><a href="monoMDS.html">scores(<i>&lt;monoMDS&gt;</i>)</a></code> <code><a href="monoMDS.html">plot(<i>&lt;monoMDS&gt;</i>)</a></code> <code><a href="monoMDS.html">points(<i>&lt;monoMDS&gt;</i>)</a></code> <code><a href="monoMDS.html">text(<i>&lt;monoMDS&gt;</i>)</a></code> </p>
        </td>
        <td><p>Global and Local Non-metric Multidimensional Scaling and
  Linear and Hybrid Scaling</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="mrpp.html">mrpp()</a></code> <code><a href="mrpp.html">meandist()</a></code> <code><a href="mrpp.html">summary(<i>&lt;meandist&gt;</i>)</a></code> <code><a href="mrpp.html">plot(<i>&lt;meandist&gt;</i>)</a></code> </p>
        </td>
        <td><p>Multi Response Permutation Procedure and Mean Dissimilarity Matrix</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="mso.html">mso()</a></code> <code><a href="mso.html">msoplot()</a></code> </p>
        </td>
        <td><p>Functions for performing and displaying a spatial partitioning
of cca or rda results</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="multipart.html">multipart()</a></code> </p>
        </td>
        <td><p>Multiplicative Diversity Partitioning</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="nestedtemp.html">nestedchecker()</a></code> <code><a href="nestedtemp.html">nestedn0()</a></code> <code><a href="nestedtemp.html">nesteddisc()</a></code> <code><a href="nestedtemp.html">nestedtemp()</a></code> <code><a href="nestedtemp.html">nestednodf()</a></code> <code><a href="nestedtemp.html">nestedbetasor()</a></code> <code><a href="nestedtemp.html">nestedbetajac()</a></code> <code><a href="nestedtemp.html">plot(<i>&lt;nestedtemp&gt;</i>)</a></code> <code><a href="nestedtemp.html">plot(<i>&lt;nestednodf&gt;</i>)</a></code> </p>
        </td>
        <td><p>Nestedness Indices for Communities of Islands or Patches</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="nobs.cca.html">nobs(<i>&lt;cca&gt;</i>)</a></code> </p>
        </td>
        <td><p>Extract the Number of Observations from a vegan Fit.</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="nullmodel.html">nullmodel()</a></code> <code><a href="nullmodel.html">print(<i>&lt;nullmodel&gt;</i>)</a></code> <code><a href="nullmodel.html">simulate(<i>&lt;nullmodel&gt;</i>)</a></code> <code><a href="nullmodel.html">update(<i>&lt;nullmodel&gt;</i>)</a></code> <code><a href="nullmodel.html">print(<i>&lt;simmat&gt;</i>)</a></code> <code><a href="nullmodel.html">smbind()</a></code> </p>
        </td>
        <td><p>Null Model and Simulation</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="oecosimu.html">oecosimu()</a></code> <code><a href="oecosimu.html">as.ts(<i>&lt;oecosimu&gt;</i>)</a></code> <code><a href="oecosimu.html">as.mcmc(<i>&lt;oecosimu&gt;</i>)</a></code> </p>
        </td>
        <td><p>Evaluate Statistics with Null Models of Biological Communities</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="ordiArrowTextXY.html">ordiArrowTextXY()</a></code> <code><a href="ordiArrowTextXY.html">ordiArrowMul()</a></code> </p>
        </td>
        <td><p>Support Functions for Drawing Vectors</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="ordiarrows.html">ordiarrows()</a></code> <code><a href="ordiarrows.html">ordisegments()</a></code> <code><a href="ordiarrows.html">ordigrid()</a></code> </p>
        </td>
        <td><p>Add Arrows and Line Segments to Ordination Diagrams</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="ordihull.html">ordihull()</a></code> <code><a href="ordihull.html">ordiellipse()</a></code> <code><a href="ordihull.html">ordibar()</a></code> <code><a href="ordihull.html">ordispider()</a></code> <code><a href="ordihull.html">ordicluster()</a></code> <code><a href="ordihull.html">summary(<i>&lt;ordihull&gt;</i>)</a></code> <code><a href="ordihull.html">summary(<i>&lt;ordiellipse&gt;</i>)</a></code> <code><a href="ordihull.html">ordiareatest()</a></code> </p>
        </td>
        <td><p>Display Groups or Factor Levels in Ordination Diagrams</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="ordilabel.html">ordilabel()</a></code> </p>
        </td>
        <td><p>Add Text on Non-transparent Label to an Ordination Plot.</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="ordiplot.html">ordiplot()</a></code> <code><a href="ordiplot.html">identify(<i>&lt;ordiplot&gt;</i>)</a></code> <code><a href="ordiplot.html">points(<i>&lt;ordiplot&gt;</i>)</a></code> <code><a href="ordiplot.html">text(<i>&lt;ordiplot&gt;</i>)</a></code> </p>
        </td>
        <td><p>Alternative plot and identify Functions for Ordination</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="ordipointlabel.html">ordipointlabel()</a></code> <code><a href="ordipointlabel.html">plot(<i>&lt;ordipointlabel&gt;</i>)</a></code> </p>
        </td>
        <td><p>Ordination Plots with Points and Optimized Locations for Text</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="ordiresids.html">ordiresids()</a></code> </p>
        </td>
        <td><p>Plots of Residuals and Fitted Values for Constrained Ordination</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="ordistep.html">ordistep()</a></code> <code><a href="ordistep.html">ordiR2step()</a></code> </p>
        </td>
        <td><p>Choose a Model by Permutation Tests in Constrained Ordination</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="ordisurf.html">ordisurf(<i>&lt;default&gt;</i>)</a></code> <code><a href="ordisurf.html">ordisurf(<i>&lt;formula&gt;</i>)</a></code> <code><a href="ordisurf.html">calibrate(<i>&lt;ordisurf&gt;</i>)</a></code> <code><a href="ordisurf.html">plot(<i>&lt;ordisurf&gt;</i>)</a></code> </p>
        </td>
        <td><p>Fit and Plot Smooth Surfaces of Variables on Ordination.</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="orditkplot.html">orditkplot()</a></code> <code><a href="orditkplot.html">plot(<i>&lt;orditkplot&gt;</i>)</a></code> <code><a href="orditkplot.html">points(<i>&lt;orditkplot&gt;</i>)</a></code> <code><a href="orditkplot.html">text(<i>&lt;orditkplot&gt;</i>)</a></code> <code><a href="orditkplot.html">scores(<i>&lt;orditkplot&gt;</i>)</a></code> </p>
        </td>
        <td><p>Ordination Plot with Movable Labels</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="orditorp.html">orditorp()</a></code> </p>
        </td>
        <td><p>Add Text or Points to Ordination Plots</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="ordixyplot.html">ordixyplot()</a></code> <code><a href="ordixyplot.html">ordisplom()</a></code> <code><a href="ordixyplot.html">ordicloud()</a></code> </p>
        </td>
        <td><p>Trellis (Lattice) Plots for Ordination</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="pcnm.html">pcnm()</a></code> </p>
        </td>
        <td><p>Principal Coordinates of Neighbourhood Matrix</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="permatfull.html">permatfull()</a></code> <code><a href="permatfull.html">permatswap()</a></code> <code><a href="permatfull.html">print(<i>&lt;permat&gt;</i>)</a></code> <code><a href="permatfull.html">summary(<i>&lt;permat&gt;</i>)</a></code> <code><a href="permatfull.html">print(<i>&lt;summary.permat&gt;</i>)</a></code> <code><a href="permatfull.html">plot(<i>&lt;permat&gt;</i>)</a></code> <code><a href="permatfull.html">lines(<i>&lt;permat&gt;</i>)</a></code> <code><a href="permatfull.html">as.ts(<i>&lt;permat&gt;</i>)</a></code> <code><a href="permatfull.html">as.mcmc(<i>&lt;permat&gt;</i>)</a></code> </p>
        </td>
        <td><p>Matrix Permutation Algorithms for Presence-Absence and Count Data</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="permustats.html">permustats()</a></code> <code><a href="permustats.html">summary(<i>&lt;permustats&gt;</i>)</a></code> <code><a href="permustats.html">densityplot(<i>&lt;permustats&gt;</i>)</a></code> <code><a href="permustats.html">density(<i>&lt;permustats&gt;</i>)</a></code> <code><a href="permustats.html">qqnorm(<i>&lt;permustats&gt;</i>)</a></code> <code><a href="permustats.html">qqmath(<i>&lt;permustats&gt;</i>)</a></code> <code><a href="permustats.html">boxplot(<i>&lt;permustats&gt;</i>)</a></code> <code><a href="permustats.html">pairs(<i>&lt;permustats&gt;</i>)</a></code> </p>
        </td>
        <td><p>Extract, Analyse and Display Permutation Results</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="permutations.html">permutations</a></code> </p>
        </td>
        <td><p>Permutation tests in Vegan</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="permutest.betadisper.html">permutest(<i>&lt;betadisper&gt;</i>)</a></code> </p>
        </td>
        <td><p>Permutation test of multivariate homogeneity of groups dispersions
  (variances)</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="plot.cca.html">plot(<i>&lt;cca&gt;</i>)</a></code> <code><a href="plot.cca.html">text(<i>&lt;cca&gt;</i>)</a></code> <code><a href="plot.cca.html">points(<i>&lt;cca&gt;</i>)</a></code> <code><a href="plot.cca.html">scores(<i>&lt;cca&gt;</i>)</a></code> <code><a href="plot.cca.html">scores(<i>&lt;rda&gt;</i>)</a></code> <code><a href="plot.cca.html">summary(<i>&lt;cca&gt;</i>)</a></code> <code><a href="plot.cca.html">print(<i>&lt;summary.cca&gt;</i>)</a></code> <code><a href="plot.cca.html">head(<i>&lt;summary.cca&gt;</i>)</a></code> <code><a href="plot.cca.html">tail(<i>&lt;summary.cca&gt;</i>)</a></code> </p>
        </td>
        <td><p>Plot or Extract Results of Constrained Correspondence Analysis
  or Redundancy Analysis</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="prc.html">prc()</a></code> <code><a href="prc.html">summary(<i>&lt;prc&gt;</i>)</a></code> <code><a href="prc.html">plot(<i>&lt;prc&gt;</i>)</a></code> </p>
        </td>
        <td><p>Principal Response Curves for Treatments with Repeated Observations</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="predict.cca.html">fitted(<i>&lt;cca&gt;</i>)</a></code> <code><a href="predict.cca.html">fitted(<i>&lt;capscale&gt;</i>)</a></code> <code><a href="predict.cca.html">residuals(<i>&lt;cca&gt;</i>)</a></code> <code><a href="predict.cca.html">predict(<i>&lt;cca&gt;</i>)</a></code> <code><a href="predict.cca.html">predict(<i>&lt;rda&gt;</i>)</a></code> <code><a href="predict.cca.html">calibrate(<i>&lt;cca&gt;</i>)</a></code> <code><a href="predict.cca.html">coef(<i>&lt;cca&gt;</i>)</a></code> <code><a href="predict.cca.html">predict(<i>&lt;decorana&gt;</i>)</a></code> </p>
        </td>
        <td><p>Prediction Tools for [Constrained] Ordination (CCA,
  RDA, DCA, CA, PCA)</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="procrustes.html">procrustes()</a></code> <code><a href="procrustes.html">summary(<i>&lt;procrustes&gt;</i>)</a></code> <code><a href="procrustes.html">plot(<i>&lt;procrustes&gt;</i>)</a></code> <code><a href="procrustes.html">points(<i>&lt;procrustes&gt;</i>)</a></code> <code><a href="procrustes.html">text(<i>&lt;procrustes&gt;</i>)</a></code> <code><a href="procrustes.html">lines(<i>&lt;procrustes&gt;</i>)</a></code> <code><a href="procrustes.html">residuals(<i>&lt;procrustes&gt;</i>)</a></code> <code><a href="procrustes.html">fitted(<i>&lt;procrustes&gt;</i>)</a></code> <code><a href="procrustes.html">predict(<i>&lt;procrustes&gt;</i>)</a></code> <code><a href="procrustes.html">protest()</a></code> </p>
        </td>
        <td><p>Procrustes Rotation of Two Configurations and PROTEST</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="pyrifos.html">pyrifos</a></code> </p>
        </td>
        <td><p>Response of Aquatic Invertebrates to Insecticide Treatment</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="radfit.html">radfit(<i>&lt;default&gt;</i>)</a></code> <code><a href="radfit.html">rad.null()</a></code> <code><a href="radfit.html">rad.preempt()</a></code> <code><a href="radfit.html">rad.lognormal()</a></code> <code><a href="radfit.html">rad.zipf()</a></code> <code><a href="radfit.html">rad.zipfbrot()</a></code> <code><a href="radfit.html">predict(<i>&lt;radline&gt;</i>)</a></code> <code><a href="radfit.html">plot(<i>&lt;radfit&gt;</i>)</a></code> <code><a href="radfit.html">plot(<i>&lt;radfit.frame&gt;</i>)</a></code> <code><a href="radfit.html">plot(<i>&lt;radline&gt;</i>)</a></code> <code><a href="radfit.html">radlattice()</a></code> <code><a href="radfit.html">lines(<i>&lt;radfit&gt;</i>)</a></code> <code><a href="radfit.html">points(<i>&lt;radfit&gt;</i>)</a></code> <code><a href="radfit.html">as.rad()</a></code> <code><a href="radfit.html">plot(<i>&lt;rad&gt;</i>)</a></code> </p>
        </td>
        <td><p>Rank -- Abundance or Dominance / Diversity Models</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="rankindex.html">rankindex()</a></code> </p>
        </td>
        <td><p>Compares Dissimilarity Indices for Gradient Detection</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="rarefy.html">rarefy()</a></code> <code><a href="rarefy.html">rrarefy()</a></code> <code><a href="rarefy.html">drarefy()</a></code> <code><a href="rarefy.html">rarecurve()</a></code> <code><a href="rarefy.html">rareslope()</a></code> </p>
        </td>
        <td><p>Rarefaction Species Richness</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="raupcrick.html">raupcrick()</a></code> </p>
        </td>
        <td><p>Raup-Crick Dissimilarity with Unequal Sampling Densities of Species</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="read.cep.html">read.cep()</a></code> </p>
        </td>
        <td><p>Reads a CEP (Canoco) data file</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="renyi.html">renyi()</a></code> <code><a href="renyi.html">plot(<i>&lt;renyi&gt;</i>)</a></code> <code><a href="renyi.html">renyiaccum()</a></code> <code><a href="renyi.html">plot(<i>&lt;renyiaccum&gt;</i>)</a></code> <code><a href="renyi.html">persp(<i>&lt;renyiaccum&gt;</i>)</a></code> </p>
        </td>
        <td><p>Renyi and Hill Diversities and Corresponding Accumulation Curves</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="reorder.hclust.html">reorder(<i>&lt;hclust&gt;</i>)</a></code> <code><a href="reorder.hclust.html">rev(<i>&lt;hclust&gt;</i>)</a></code> <code><a href="reorder.hclust.html">scores(<i>&lt;hclust&gt;</i>)</a></code> <code><a href="reorder.hclust.html">cutreeord()</a></code> </p>
        </td>
        <td><p>Reorder a Hierarchical Clustering Tree</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="scores.html">scores(<i>&lt;default&gt;</i>)</a></code> </p>
        </td>
        <td><p>Get Species or Site Scores from an Ordination</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="screeplot.cca.html">screeplot(<i>&lt;cca&gt;</i>)</a></code> <code><a href="screeplot.cca.html">screeplot(<i>&lt;decorana&gt;</i>)</a></code> <code><a href="screeplot.cca.html">screeplot(<i>&lt;prcomp&gt;</i>)</a></code> <code><a href="screeplot.cca.html">screeplot(<i>&lt;princomp&gt;</i>)</a></code> <code><a href="screeplot.cca.html">bstick()</a></code> </p>
        </td>
        <td><p>Screeplots for Ordination Results and Broken Stick Distributions</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="simper.html">simper()</a></code> <code><a href="simper.html">summary(<i>&lt;simper&gt;</i>)</a></code> </p>
        </td>
        <td><p>Similarity Percentages</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="simulate.rda.html">simulate(<i>&lt;rda&gt;</i>)</a></code> </p>
        </td>
        <td><p>Simulate Responses with Gaussian Error or Permuted Residuals for Constrained Ordination</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="sipoo.html">sipoo</a></code> <code><a href="sipoo.html">sipoo.map</a></code> </p>
        </td>
        <td><p>Birds in the Archipelago of Sipoo (Sibbo and Borgå)</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="spantree.html">spantree()</a></code> <code><a href="spantree.html">as.hclust(<i>&lt;spantree&gt;</i>)</a></code> <code><a href="spantree.html">cophenetic(<i>&lt;spantree&gt;</i>)</a></code> <code><a href="spantree.html">spandepth()</a></code> <code><a href="spantree.html">plot(<i>&lt;spantree&gt;</i>)</a></code> <code><a href="spantree.html">lines(<i>&lt;spantree&gt;</i>)</a></code> </p>
        </td>
        <td><p>Minimum Spanning Tree</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="specaccum.html">specaccum()</a></code> <code><a href="specaccum.html">plot(<i>&lt;specaccum&gt;</i>)</a></code> <code><a href="specaccum.html">boxplot(<i>&lt;specaccum&gt;</i>)</a></code> <code><a href="specaccum.html">fitspecaccum()</a></code> <code><a href="specaccum.html">plot(<i>&lt;fitspecaccum&gt;</i>)</a></code> <code><a href="specaccum.html">predict(<i>&lt;specaccum&gt;</i>)</a></code> <code><a href="specaccum.html">predict(<i>&lt;fitspecaccum&gt;</i>)</a></code> <code><a href="specaccum.html">specslope()</a></code> </p>
        </td>
        <td><p>Species Accumulation Curves</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="specpool.html">specpool()</a></code> <code><a href="specpool.html">estimateR()</a></code> <code><a href="specpool.html">specpool2vect()</a></code> <code><a href="specpool.html">poolaccum()</a></code> <code><a href="specpool.html">estaccumR()</a></code> <code><a href="specpool.html">summary(<i>&lt;poolaccum&gt;</i>)</a></code> <code><a href="specpool.html">plot(<i>&lt;poolaccum&gt;</i>)</a></code> </p>
        </td>
        <td><p>Extrapolated Species Richness in a Species Pool</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="sppscores.html">`sppscores<-`()</a></code> </p>
        </td>
        <td><p>Add or Replace Species Scores in Distance-Based Ordination</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="stepacross.html">stepacross()</a></code> </p>
        </td>
        <td><p>Stepacross as Flexible Shortest Paths or Extended Dissimilarities</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="stressplot.wcmdscale.html">stressplot(<i>&lt;wcmdscale&gt;</i>)</a></code> </p>
        </td>
        <td><p>Display Ordination Distances Against Observed Distances in Eigenvector Ordinations</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="taxondive.html">taxondive()</a></code> <code><a href="taxondive.html">taxa2dist()</a></code> </p>
        </td>
        <td><p>Indices of Taxonomic Diversity and Distinctness</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="tolerance.html">tolerance()</a></code> </p>
        </td>
        <td><p>Species tolerances and sample heterogeneities</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="treedive.html">treedive()</a></code> <code><a href="treedive.html">treeheight()</a></code> <code><a href="treedive.html">treedist()</a></code> </p>
        </td>
        <td><p>Functional Diversity and Community Distances from Species Trees</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="tsallis.html">tsallis()</a></code> <code><a href="tsallis.html">tsallisaccum()</a></code> <code><a href="tsallis.html">persp(<i>&lt;tsallisaccum&gt;</i>)</a></code> </p>
        </td>
        <td><p>Tsallis Diversity and Corresponding Accumulation Curves</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="varechem.html">varechem</a></code> <code><a href="varechem.html">varespec</a></code> </p>
        </td>
        <td><p>Vegetation and environment in lichen pastures</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="varpart.html">varpart()</a></code> <code><a href="varpart.html">showvarparts()</a></code> <code><a href="varpart.html">plot(<i>&lt;varpart234&gt;</i>)</a></code> </p>
        </td>
        <td><p>Partition the Variation of Community Matrix by 2, 3, or 4 Explanatory Matrices</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="vegan-deprecated.html">vegan-deprecated</a></code> </p>
        </td>
        <td><p>Deprecated Functions in vegan package</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="vegan-internal.html">ordiParseFormula()</a></code> <code><a href="vegan-internal.html">ordiTerminfo()</a></code> <code><a href="vegan-internal.html">ordiNAexclude()</a></code> <code><a href="vegan-internal.html">ordiNApredict()</a></code> <code><a href="vegan-internal.html">ordiArgAbsorber()</a></code> <code><a href="vegan-internal.html">centroids.cca()</a></code> <code><a href="vegan-internal.html">getPermuteMatrix()</a></code> <code><a href="vegan-internal.html">howHead()</a></code> <code><a href="vegan-internal.html">pasteCall()</a></code> <code><a href="vegan-internal.html">veganCovEllipse()</a></code> <code><a href="vegan-internal.html">veganMahatrans()</a></code> <code><a href="vegan-internal.html">hierParseFormula()</a></code> <code><a href="vegan-internal.html">GowerDblcen()</a></code> <code><a href="vegan-internal.html">addLingoes()</a></code> <code><a href="vegan-internal.html">addCailliez()</a></code> </p>
        </td>
        <td><p>Internal vegan functions</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="vegan-package.html">vegan-package</a></code> </p>
        </td>
        <td><p>Community Ecology Package: Ordination, Diversity and Dissimilarities</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="vegdist.html">vegdist()</a></code> </p>
        </td>
        <td><p>Dissimilarity Indices for Community Ecologists</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="vegemite.html">vegemite()</a></code> <code><a href="vegemite.html">tabasco()</a></code> <code><a href="vegemite.html">coverscale()</a></code> </p>
        </td>
        <td><p>Display Compact Ordered Community Tables</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="wascores.html">wascores()</a></code> <code><a href="wascores.html">eigengrad()</a></code> </p>
        </td>
        <td><p>Weighted Averages Scores for Species</p></td>
      </tr><tr>
        
        <td>
          <p><code><a href="wcmdscale.html">wcmdscale()</a></code> <code><a href="wcmdscale.html">plot(<i>&lt;wcmdscale&gt;</i>)</a></code> <code><a href="wcmdscale.html">scores(<i>&lt;wcmdscale&gt;</i>)</a></code> </p>
        </td>
        <td><p>Weighted Classical (Metric) Multidimensional Scaling</p></td>
      </tr>
    </tbody>
    </table>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="sidebar">
    <h2>Contents</h2>
    <ul class="nav nav-pills nav-stacked">
      <li><a href="#section-all-functions">All functions</a></li>
    </ul>
  </div>
</div>


      <footer>
      <div class="copyright">
  <p>Developed by Jari Oksanen.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.4.1.</p>
</div>

      </footer>
   </div>

  


  </body>
</html>


